src/benchmark.rs

```diff
@@ -90,6 +90,28 @@ impl AsvCompare {
     }
 }
 
+pub async fn resolve_env(wd: &Path) -> Result<Vec<String>> {
+    tracing::info!("Resolving Environments: {:?}", wd);
+    let env = resolve_env_from_stdout(
+        Command::new("python")
+            .current_dir(wd)
+            .args([
+                "-c",
+                "import asv; import json; conf = asv.config.Config.load(\"asv.conf.json\"); env_names = [env.name for env in asv.environment.get_environments(conf, \"\")]; print(json.dumps(env_names))"
+            ]),
+    )
+    .await?;
+    Ok(env)
+}
+
+async fn resolve_env_from_stdout(command: &mut Command) -> Result<Vec<String>> {
+    let stdout_env_specs_buffer = command.output().await?.stdout;
+    let stdout_env_specs = String::from_utf8(stdout_env_specs_buffer)?;
+    let parsed: Vec<String> = serde_json5::from_str(&stdout_env_specs)?;
```

Comment:
```suggestion
    let parsed: Vec<String> = serde_json::from_str(&stdout_env_specs)?;
```

Reply:
huh, I thought I added it. Well, adding it will be a +2-0 diff (one line in lockfile and cargo.toml) since we already transitively depend on it.